### DISCUSSION TITLE
Support Ring for Individual Sides

### DISCUSSION THREAD

**Discussion by prince272** [tailwindlabs/tailwindcss#11411] [Help]
Please I need help to support Ring for Individual Sides like border-r-1 border-r-0 border-l-0

**Comment by wongjn** [comment]
The ring utilities use `box-shadow`, so you'd want to look at creating your own `box-shadow`s like https://play.tailwindcss.com/voMd03Wqil. You could then look at [writing a Tailwind plugin](https://tailwindcss.com/docs/plugins#adding-utilities) to have these `box-shadow`s as utility classes.

**Reply by prince272** [reply]
Could you please assist me in incorporating a plugin to achieve specific behaviors like the order.

I am attempting to construct an input group that resembles the example provided. When the input is focused, I would like the border color to be 2px. Currently, the default state displays a border with 1px thickness, and upon focusing, the border turns blue with a 1px. https://www.tailwindreact.com/application-ui/forms/input-groups? Input group with embedded icon decoration on the left

I want the ring for individual slides so I can apply roundness properly.

**Reply by wongjn** [reply]
> Could you please assist me in incorporating a plugin to achieve specific behaviors like the one demonstrated on this URL: https://www.tailwindreact.com/application-ui/forms/input-groups?

Unfortunately not, that seems a bit out of scope for me. Consider providing a [Tailwind Play](https://play.tailwindcss.com) of the DOM you have and I could at least get you started.

**Reply by prince272** [reply]
okay. Thanks for your response. I really appreciate it.
